tests/e2e/runner/__tests__/minter-suite-v2/i-shared-minter-v0.test.ts

```
@@ -119,17 +129,59 @@ describe("iSharedMinterV0 event handling", () => {
     });
   });
 
-  // TODO - need ERC20 minter to test event handling of "ProjectCurrencyInfoUpdated"
-  // ERC20 minter is not yet developed at the time of writing this test
-  // describe("ProjectCurrencyInfoUpdated", () => {
-  //   afterEach(async () => {
-  //     // TODO
-  //   });
-
-  //   test("Currency is updated and configured", async () => {
-  //     // TODO - need ERC20 minter to test this
-  //   });
-  // });
+  describe("ProjectCurrencyInfoUpdated", () => {
+    afterEach(async () => {
+      // clear the minter for project zero
+      try {
+        await sharedMinterFilterContract
+          .connect(artist)
+          .removeMinterForProject(0, genArt721CoreAddress);
+      } catch (error) {
+        // swallow error in case of test failure
```

swallow error in case of test failure